src/aeos/owl_dune.omp

```diff
@@ -0,0 +1,201 @@
+(copy_files# ext/*)
+(copy_files# core/*)
+
+; copy core/openmp/* with the exception of dummy file
```

Comment:
Instead of creating a new file, you can conditionally generate this section and enable aeos when ENABLE_OPENMP (or even better OWL_ENABLE_OPENMP) is set by using the ocaml syntax in dune, e.g. https://github.com/xapi-project/xen-api/blob/master/ocaml/xapi/dune#L2

This would be much cleaner than keeping all this copies of stuff lying around and remembering to always keep track of them.

Reply:
Depending on the variable you can write in the dune file to either copy the dummy header or the generated ones. Do not set the variable in the dune file as done now (in fact, remove all those lines as they are kind of unneeded now), so that people calling `OWL_ENABLE_OPENMP=1 opam install owl` or `OWL_ENABLE_OPENMP=1 dune build` will have the `aeos` version automatically

Reply:
Thanks you so much for your support and help @mseri. After some thoughts on this issue, I think the configuration of AEOS itself is not very well thought of from the beginning. What you have proposed is indeed a much better and systematic solution. However, to reduce the "attack surface" in the Owl configuration issue, I believe it is cleaner for now to just totally disable AEOS from the dune file. Right now I'm bit occupied, and I plan to come back later to give a full review of this module and its configuration.

Reply:
As you wish but I think it is actually easier to simply generate the empty file